src/components/views/elements/SSOButtons.tsx

```diff
@@ -58,7 +58,7 @@ const SSOButton: React.FC<ISSOButtonProps> = ({
     mini,
     ...props
 }) => {
-    const label = idp ? _t("Continue with %(provider)s", { provider: idp.name }) : _t("Sign in with single sign-on");
+    const label = idp ? _t("Continue with %(provider)s", { provider: idp.name }) : ("Log in with single sign-on");
```

Comment:
This breaks internationalisation in element. I'd suggest updating the value in src\i18n\strings\en_EN.json